src/nrnoc/code.cpp

```diff
@@ -3,4 +3,4 @@
  * Automake doesn't deal well with sources that live in other directories, so
  * this is a quick and dirty workaround.
  */
-#include "../oc/hocusr.c"
+#include "../oc/code.cpp"
```

Comment:
curious - reason for renaming from hocusr.c to code.cpp was?

Reply:
Don't remember, but this way of including `.cpp` into `.cpp` files was done for autotools limitations.  Needs to be refactored.

Reply:
Ok